jordan_bundy/README.md

```diff
@@ -0,0 +1,17 @@
+# My Assignment
```

Comment:
Good readme!  Generally you don't need to include info on tests, since they are a development tool and not usually used "in the wild" when people download your package.